tests/evm/test_callop.py

```diff
@@ -129,15 +129,12 @@ def gen_testing_data():
     ]
     stacks = [
         Stack(),
-        Stack(value=int(1e18)),
-        Stack(gas=100),
-        Stack(gas=100000),
-        Stack(cd_offset=64, cd_length=320, rd_offset=0, rd_length=32),
-        Stack(cd_offset=0, cd_length=32, rd_offset=64, rd_length=320),
```

Comment:
Can you expand on why we are removing these tests?

Reply:
There are 768 test cases in `test_callop` now and that's why I merged these 5 different stacks into 2 to reduce the number of cases. There will be 1,344 cases if I don't merge them. But the most important is I think it still covers the testing scenarios or scope as it was.